common.props

```diff
@@ -3,7 +3,7 @@
     <ContinuousIntegrationBuild Condition="'$(TF_BUILD)' == 'true'">true</ContinuousIntegrationBuild>
 
     <MajorProductVersion>4</MajorProductVersion>
-    <MinorProductVersion>5</MinorProductVersion>
+    <MinorProductVersion>6</MinorProductVersion>
```

Comment:
We are bumping only minor version here, but we are doing a major version bump for `Newtonsoft.Json` (11.X to 13.X). Is that intentional? Should we not also bump the major version of this package (Microsoft.Net.Sdk.Functions) ?

Reply:
The host already loads 13.x for in-proc and this change is not breaking (tested locally w/ SDK and `Newtonsoft.Json` 13.x). @jviau also noted that `Newtonsoft.Json` does not really follow semver.